src/components/shared/MarkerTooltipContents.js

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import React, { PureComponent } from 'react';
+import classNames from 'classnames';
+import formatTimeLength from '../../utils/format-time-length';
+
+import type { TracingMarker } from '../../types/profile-derived';
+import type { MarkerPayload } from '../../types/markers';
+
+function getMarkerDetails(data: MarkerPayload): React$Element<*> | null {
+  if (data) {
+    switch (data.type) {
+      case 'UserTiming': {
+        return (
+          <div className="tooltipDetails">
+            <div className="tooltipLabel" key="name">
+              Name:
+            </div>
+            {data.name}
+          </div>
+        );
+      }
```

Comment:
This is an appropriate place where the display can be enriched for other marker types in the future